assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -84,13 +108,31 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
+		case SET_CAPABILITIES: {
+			const { capabilities } = payload;
+
+			return {
+				...state,
+				capabilities,
+			};
+		}
+
 		default: {
 			return { ...state };
 		}
 	}
 };
 
-export const resolvers = {};
+export const resolvers = {
+	/**
+	 * Fullfills user capabilities on initial request.
+	 *
+	 * @since n.e.x.t
+	 */
+	*getCapabilities() {
+		yield actions.setCapabilities( global._googlesitekitUserData?.permissions );
```

Comment:
Let's be more strict here, if this object is not set, we should `global.console.error( 'Could not load core/user permissions.' )` and return early.

Reply:
Updated.